.travis.yml

```diff
@@ -4,14 +4,14 @@ scala:
 jdk:
 - oraclejdk8
 before_install:
-- if [ "$TRAVIS_PULL_REQUEST" = "false" ]; then openssl aes-256-cbc -K $encrypted_86276aecec54_key -iv $encrypted_86276aecec54_iv
-  -in secring.gpg.enc -out secring.gpg -d; fi
+- if [ "$TRAVIS_PULL_REQUEST" = "true" ]; then bash
```

Comment:
This check condition won't work once this PR is merged in master (the openssl command doesn't work for PR forks. Therefore, 

```if [ "$TRAVIS_BRANCH" = "master" -a "$TRAVIS_PULL_REQUEST" = "false" ]; then bash ...```